doc/doxygen-awesome-css/download_stylesheet.sh

```diff
@@ -0,0 +1,10 @@
+#! /usr/bin/env bash
```

Comment:
Why is this file here? Wouldn't it make sense to include all needed files here (that is at least what I do elsewhere)?

Reply:
All files needed are included in the repository. This file can be removed.

The hope is that this file documents which files need to be downloaded to update the styling. The script provides the answer, just one, and it also provides the URL where that one file can be found (hopefully).

Would you like me to remove it?

Reply:
My suggestion would be to have a GH Action check if doxygen-awesome is outdated. It could run for example on all merges to the main branch. 
Ideally it could even open a PR if it founds things that are outdated, but I don't know if that is possible.

Reply:
Just the first hit when searching : https://github.com/istein/gh-actions-open-pr/blob/main/.github/workflows/pr.yml 
It seems like we can modify this to do what we want

Reply:
Nice idea, I'll improve the PR to incorporate this.

Reply:
Amazing. Let me know if you need help!